packages/wasm-utxo/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bitgo/wasm-utxo",
   "description": "WebAssembly wrapper for rust-bitcoin (beta)",
-  "version": "0.0.0-semantic-release-managed",
+  "version": "0.0.1",
```

Comment:
@OttoAllmendinger keeping track of the version number in the `package.json` file will require creating a new commit during the release, which will significantly increase the complexity of the release workflow